reed-solomon-gao-mateer/src/util.rs

```diff
@@ -0,0 +1,59 @@
+pub const fn log2(mut x: usize) -> usize {
```

Comment:
Why not `ilog2`? It should be implemented somehow via `ffs` (or `[leading_zeros](https://doc.rust-lang.org/std/primitive.usize.html#method.leading_zeros)`).

Reply:
Because it's still WIP and only at th level what I did 2 years ago, so it's not getting lost if upstream yanks the branch :) still busy with the SIMD